api/incubator/src/test/java/io/opentelemetry/api/incubator/logs/ExtendedLogsBridgeApiUsageTest.java

```diff
@@ -76,4 +87,148 @@ void loggerEnabled() {
+        .setAttribute(doubleKey, 1.1)
+        .setAttribute(strArrKey, Arrays.asList("value1", "value2"))
+        .setAttribute(longArrKey, Arrays.asList(1L, 2L))
+        .setAttribute(booleanArrKey, Arrays.asList(true, false))
+        .setAttribute(doubleArrKey, Arrays.asList(1.1, 2.2))
+        .setAttribute(
+            mapKey,
+            ExtendedAttributes.builder().put("childStr", "value").put("childLong", 1L).build())
+        .setAllAttributes(Attributes.builder().put("key1", "value").build())
+        .setAllAttributes(ExtendedAttributes.builder().put("key2", "value").build())
+        .emit();
+
+    assertThat(exporter.getFinishedLogRecordItems())
+        .satisfiesExactly(
+            logRecordData -> {
+              assertThat(logRecordData).isInstanceOf(ExtendedLogRecordData.class);
+              ExtendedLogRecordData extendedLogRecordData = (ExtendedLogRecordData) logRecordData;
+
+              // Optionally access standard attributes, which filters out any extended attribute
+              // types
+              assertThat(extendedLogRecordData.getAttributes())
+                  .isEqualTo(
+                      Attributes.builder()
+                          .put(strKey, "value")
+                          .put(longKey, 1L)
+                          .put(booleanKey, true)
+                          .put(doubleKey, 1.1)
+                          .put(strArrKey, Arrays.asList("value1", "value2"))
+                          .put(longArrKey, Arrays.asList(1L, 2L))
+                          .put(booleanArrKey, Arrays.asList(true, false))
+                          .put(doubleArrKey, Arrays.asList(1.1, 2.2))
+                          .put("key1", "value")
+                          .put("key2", "value")
+                          .build());
+
+              // But preferably access and serialize full extended attributes
+              assertThat(extendedLogRecordData.getExtendedAttributes())
+                  .isEqualTo(
+                      ExtendedAttributes.builder()
+                          .put(strKey, "value")
+                          .put(longKey, 1L)
+                          .put(booleanKey, true)
+                          .put(doubleKey, 1.1)
+                          .put(strArrKey, Arrays.asList("value1", "value2"))
+                          .put(longArrKey, Arrays.asList(1L, 2L))
+                          .put(booleanArrKey, Arrays.asList(true, false))
+                          .put(doubleArrKey, Arrays.asList(1.1, 2.2))
+                          .put(
+                              mapKey,
+                              ExtendedAttributes.builder()
+                                  .put("childStr", "value")
+                                  .put("childLong", 1L)
+                                  .build())
+                          .put("key1", "value")
+                          .put("key2", "value")
+                          .build());
+            });
+  }
```

Comment:
@jack-berg do we know of cases that are allowed in the spec that we don't yet cover? And if so, can we get those documented somewhere? I haven't looked to verify, but off the top of my head:
* lists of lists?
* heterogeneous lists (content of varying types)
* lists of maps?

Reply:
Yeah this question needs more work at the spec / semantic convention level.

I would say that given that according to the spec, log attributes are [map<string, any>](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-attributes), all of those types you've listed are valid `any` values. 

With that said, I know there is some desire to see if we can limit log attributes to only supporting standard attributes PLUS maps of maps. See convo [here](https://github.com/open-telemetry/opentelemetry-java/pull/7123#discussion_r1962550976). Easier to add missing things later than to go to far and then try to backtrack.

Reply:
I was just hoping we could make it easier for users or otel devs to quickly find what's not supported....but I suppose it's fine to just point at the spec.